index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Event Site</title>
+  </head>
+  <body>
+    <nav class="site-nav js-toggle-menu">
+      <button
+        type="button"
+        class="u-button-reset menu-trigger js-toggle-trigger"
+        aria-controls="site-menu"
+      >
+        <span aria-hidden="true" class="menu-icon">
+          <div class="menu-icon__line"></div>
+          <div class="menu-icon__line"></div>
+          <div class="menu-icon__line"></div>
+        </span>
+        Menu
+      </button>
+      <ul id="site-menu" class="u-list-reset menu-list">
+        <!-- <ul id="site-menu" class="u-list-reset menu-list full-screen"> -->
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+    <h1>Decolonizing Artistic Hiring Practices</h1>
+    <p>
+      A conference for the advancement of inclusive, anti-racist, equal, and
+      diverse hiring processes within the arts sector. For Employers,
+      Businesses, and Organisations seeking to make a difference in their
+      community.
+    </p>
+    <p>
+      120 W North Ave #302, Baltimore, MD 21201 November 20, 2020 - 1:15 pm
+    </p>
+    <h2>
+      Panelists
+    </h2>
+    <img
+      src="https://cdn.glitch.com/cfdb336d-7569-49ae-8522-adb0805e7f54%2FMichael.jpg?v=1607559667627"
+      alt="A man with brown skin wearing a red shirt with his arms crossed and smiling towards the camera"
+    />
+    <p>
+      Michael J. Bobbit
+    </p>
```

Comment:
**Not** required, but the speaker name and/or the talk name could also be worthy of headings (`h3`).